sapi/cli/tests/php_cli_server_014.phpt

```diff
@@ -61,12 +61,11 @@ X-Powered-By: %s
 Content-type: %s
 
 done
-HTTP/1.1 404 Not Found
+HTTP/1.1 200 OK
 Host: %s
 Date: %s
 Connection: close
-Content-Type: %s
-Content-Length: %d
+X-Powered-By: PHP/%s
```

Comment:
a little related, I would expect all pages, even 404 to return `X-Powered-By: PHP/%s` when served with php server